.travis.yml

```diff
@@ -114,7 +114,7 @@ install:
   # Prepare directories
   - "mkdir -p $TRAVIS_BUILD_DIR/build/scripts"
   - "cd $TRAVIS_BUILD_DIR/build/scripts"
-  - "mkdir -p $TRAVIS_BUILD_DIR/build/release"
+  - "mkdir -p $TRAVIS_BUILD_DIR/build/Release"
```

Comment:
nit, on unix, the paths are typically lower-case

Reply:
>  I will put in platform conditional switches. 

No, don't do that. 
I thought you've just changed unintentionally, if Windows expects only Capitalfirst, then Debug & Release are fine and easy!